src/arrayMethodJoin.js

```diff
@@ -5,7 +5,32 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let result = '';
+    let sep = separator;
+
+    if (sep === undefined) {
+      sep = ',';
+    }
```

Comment:
I think you forgot that you can assign a dafault value. Example: function(separator = ',')